### DISCUSSION TITLE
How is the github README screenshots setup?

### DISCUSSION THREAD

**Discussion by patrickacollier** [blakeblackshear/frigate#12579] [Ask A Question] [answered]
### What is your question:

so i have frigate setup in hassOS and so the frigate card is using the standard ingress, but it doesn't look anything like the demo page. 2 main questions
1. what magic are these screenshots? Is there a tutorial out there?
2. can you access frigate (on hassOS) outside the ingress (i realize this is unprotected)

The multiCamera scrubbing view looks awesome.

My current system version (SYSTEM 0.13.2-6476F8A)

## My view
![image](https://github.com/user-attachments/assets/249cf3f4-7d35-4674-8cdb-27e4d10936be)

## Github screenshot

![Demo Screenshot](https://private-user-images.githubusercontent.com/569905/337930841-5e713cb9-9db5-41dc-947a-6937c3bc376e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE3NzAyMzAsIm5iZiI6MTcyMTc2OTkzMCwicGF0aCI6Ii81Njk5MDUvMzM3OTMwODQxLTVlNzEzY2I5LTlkYjUtNDFkYy05NDdhLTY5MzdjM2JjMzc2ZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyM1QyMTI1MzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xYjcwNWVjNTRiNjA4YzVhZjBhZjljMTMyNTJkMzZiMWQ4ODA3YTZjNmRkZWQ1MjI3OTc1MWRlMTFkNDBmNmMwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.XPHkILY9gaRGYdqJQcXNptc9IflepvLddO1JeUR_k6A)

**Comment by NickM-27** [comment, marked as answer]
1. this is Frigate 0.14 which is currently in beta (4), likely will be a release candidate soon. 
2. yes, just need to enable a port, though it is worth noting frigate 0.14 supports authentication natively

**Reply by patrickacollier** [reply]
ah, ok that explains it. It looks great. Thanks for the info.

so my current integration shows 'ccab4aaf-frigate:5000' does this indicate the port is already open?
I'm only just now getting wall panels setup and would like the flexibility of using it outside of ingress.

**Reply by NickM-27** [reply]
No, that is using the ingress via HA, you'd need to open the port so you can connect via home assistant ip

**Reply by patrickacollier** [reply]
Got it! found it in add-ons config. 
Also i was somehow on the DEV branch of github instead of master. likely how I got shocked by the screenshots. 
Looking forward to this for sure.
